src/test/java/com/puppycrawl/tools/checkstyle/api/FullIdentTest.java

```diff
@@ -241,4 +244,15 @@ public void testFullyQualifiedStringArray() throws Exception {
                 .isEqualTo(String[].class.getCanonicalName());
     }
 
+    @Test
+    public void testCreateFullIdentBelow2() throws Exception {
+        final DefaultConfiguration checkConfig = createModuleConfig(ImportOrderCheck.class);
+        final String[] expected = {
+            "3:1: " + getCheckMessage(ImportOrderCheck.class,
+                    MSG_ORDERING, "java.util.HashMap"),
+        };
+
+        verify(checkConfig, getPath("InputFullIdent.java"),
+                expected);
+    }
```

Comment:
Why haven't we used the InlineConfigParser?

Reply:
done removed verify method.

Reply:
Resolved